### ISSUE TITLE
Create Reviewers Guide

### ISSUE THREAD

**Issue by negin513** [UCAR-SEA/SEA-ISS-Template#2]

**Comment by kafitzgerald** [comment]
From our discussion today:

Would be great to make this templated and visible to submitters as well.  We could look at JOSS and SciPy for some examples.

**Comment by negin513** [comment]
# ✅ Suggested Review Criteria for Notebook Proceedings

These criteria are intended to support authors and reviewers of notebook-based submissions. Feedback and improvements are welcome via PR or discussion.

## ⚖️ Conflict of Interest
- [ ] As the reviewer, I confirm that there are **no conflicts of interest** for me to review this work  
(If you are unsure whether you are in conflict, please speak to your editor _before_ starting your review.)

## 🧾 Overall Quality
- [ ] JupyterBook is self-contained and readable top-to-bottom
- [ ] Includes a short abstract or introduction
- [ ] Notebook length and complexity are appropriate for the workshop
- [ ] Final product is useful to others in the community

## 📖 Written Quality
- [ ] Text is written clearly in English
- [ ] Narrative is well-structured (consistent voice, tense, flow)
- [ ] Jargon is avoided or clearly explained
- [ ] The content furthers the reader’s understanding or research

## 🧠 Technical Content
- [ ] Scientific reasoning is sound and methodologically appropriate
- [ ] Code cells are relevant to the content
- [ ] Clear explanation of algorithms or methods

## 🚀 Novelty & Impact
- [ ] Demonstrates a new method, tool, or insight for the field
- [ ] Contribution advances scientific computing or domain-specific workflows
- [ ] Clear explanation of why this matters and for whom

## 🧑‍💻 Code Quality
- [ ] If code snippets are included, the code is readable, modular, and commented where necessary
- [ ] Variable/function names are meaningful
- [ ] Avoids unnecessary or hidden state between cells
- [ ] No leftover debugging code or redundant outputs

## 📊 Figures and Tables
- [ ] Visualizations are appropriate and enhance understanding
- [ ] All figures & Tables have captions and readable labels

## 📄 Citations and Licensing
- [ ] All referenced software/tools are properly cited
- [ ] License and reuse conditions are respected (figures, code, data)
- [ ] Mathematical terms and units are clearly defined
- [ ] Acronyms are spelled out at first use
- [ ] All contributing authors are listed (e.g., in the README or notebook header)

## ✍️ Reviewer Suggestions
Please summarize:
- Key strengths of the submission
- Any recommended improvements
- Whether it should be accepted as-is, with revisions, or not accepted

**Comment by negin513** [comment]
Some suggested labels: 

## 🏷️ Review Labeling Guidelines

Please apply the appropriate labels to help track the status of the submission:

- `under-review` — Reviewer has started reviewing the submission
- `review-completed` — Reviewer has completed their evaluation and submitted feedback
- `minor-revisions` — Revisions are needed, but the submission is generally sound
- `major-revisions` — Substantial updates are required before reconsideration
- `accepted` — Submission has been accepted after review
- `rejected` — Submission has been declined

**Comment by vanderwb** [comment]
This looks like a good start to me. I have the following thoughts/suggestions:

1. I think we could probably leave out the Novelty section, as we happily allow talk submissions that simply highlight best practices. These need not be novel, and so I don't know that we should maintain such a criteria for the proceedings either.
2. The "Code Quality" section could probably go into the technical review, along with some basic functionality checks like "Environment resolves (if used)" and "Cells successfully execute". Everything else above looks good for the content review.
3. We might want to clarify which the above are **required** for acceptance. Do we want all boxes checked before accepting?
4. I don't know that "Notebook length and complexity are appropriate for the workshop" will be useful without prior examples to go off of. Perhaps we drop it for this year and once we have a baseline, add it as criteria next time.
5. Instead of "Scientific reasoning..." in technical content, I'd suggest simply "Approach..."

**Comment by negin513** [comment]
Thanks for the comments @vanderwb . Edited it to address the above. 

For us, once the submission is in:

- [ ] Add label under review
- [ ] Contact user to transfer the ownership to UCAR-SEA
- [ ] Check if JB is building and all smaller components are there.

Once these are in place, pass them around to the reviewers to review.

**Comment by kafitzgerald** [comment]
Thanks so much for putting this together.  This looks great!

I think my only concern is that the labels might add some unnecessary complexity and confusion especially with multiple reviewers.  Perhaps we could thin the list down to "under-review" (to indicate the review process is underway) and "accepted" (to indicate it's complete and accepted) and then add others as needed.

**Comment by vanderwb** [comment]
Agree with @kafitzgerald 's suggestion, but I wonder if it still might be helpful to have one revision category. Something like "revision-pending" to indicate we are waiting on the author response. If the expectation is that the submitter will create a new issue for the revision, then that wouldn't be useful, however.
